README.rst

```diff
@@ -0,0 +1,23 @@
+GENETIS_HPol
+============
+
+The genetic algorithm (GA) is a submodule 
+(links to Shared-Code/GA; branch HPol; commit "HPol pause")
+
+
+GENETIS_doc
+-----------
+This directory contains the documentation.
+
+To view the documentation, please use this `link
+<https://genetis-doc.vercel.app/>`_.
+
+To edit the `rst` files locally with Sphinx, note that 
```

Comment:
This can be automated in the `Makefile`, so that when you do `make html` it creates a venv, installs the deps, and builds the docs.

Reply:
In that case, would the user be `pip install`ing the required python packages every time the html files are build using the Makefile?

Reply:
Not, the `venv` will stick around with the packages already installed within it, without affecting/installing the packages globally.  If you want to attempt this let me know -- I can point you to some repos that follow this approach.